doc/python_api_reference_vDev.md

```diff
@@ -7546,6 +7547,34 @@ class TableauSimulator:
     """
 ```
 
+<a name="stim.TableauSimulator.__init__"></a>
+```python
+# stim.TableauSimulator.__init__
+
+# (in class stim.TableauSimulator)
+def __init__(
+    self,
+    *,
+    seed: Optional[int] = None,
+) -> None:
+
+    """Initializes a stim.TableauSimulator.
+
+    Args:
+        seed: Optional seed for the pseudo-random number generator.
```

Comment:
Copy the differs-across-versions warning from the other seed arguments.

Reply:
Done.